packages/api/src/constants.js

```diff
@@ -5,6 +5,7 @@ let genesisTime
 module.exports = {
   EPOCH_CHANGE_TIME: Number(process.env.EPOCH_CHANGE_TIME),
   TCP_CONNECT_TIMEOUT: Number(process.env.TCP_CONNECT_TIMEOUT),
+  DPNS_CONTRACT: process.env.DPNS_CONTRACT ?? '5mjGWa9mruHnLBht3ntbfgodcSoJxA1XIfYiv1PFMVU=',
```

Comment:
I think we can just hardcode it, it does not change across networks

Reply:
✅